docusaurus.config.js

```diff
@@ -103,6 +117,12 @@ const config = {
                         position: "left",
                         label: "Docs",
                     },
+                    {
+                        to: '/community/community_resources',
+                        label: 'Community Guide',
```

Comment:
```suggestion
                        label: 'Community',
```

Reply:
Doing it just to keep it same to the original label. `Community` sounds better in the top bar.